index.js

```diff
@@ -18,7 +12,53 @@ function displayBooks(book, index) {
   return content;
 }
 
-function storeForm(form) {s
+function storeForm(form) {
   localStorage.setItem('form', JSON.stringify(form));
 }
-s
\ No newline at end of file
+
+const addButton = document.getElementById('addBook');
+const newTitle = document.getElementById('bookTitle');
+const newAuthor = document.getElementById('authorName');
+const container = document.querySelector('.bodyh1');
+
+addButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (newAuthor.value !== '' && newTitle.value !== '') {
+    const title = newTitle.value;
+    const author = newAuthor.value;
+    const newBook = {
+      title,
+      author,
+    };
+
+    books.push(newBook);
+    storeForm(books);
+    const book = displayBooks(newBook, books.length - 1);
+
+    container.innerHTML += book;
+  }
+  addButton.form.reset(); // use form property of the button element to reset the form
+});
+
+for (let i = 0; i < books.length; i += 1) {
+  const book = displayBooks(books[i], i);
+  container.innerHTML += book;
+}
+
+const deleteBtn = document.querySelectorAll('[id^="delete"]');
+deleteBtn.forEach((button) => {
+  button.addEventListener('click', () => {
+    const item = button.parentNode;
+    item.parentNode.removeChild(item);
+
+    const { id } = button;
+
+    for (let i = 0; i < books.length; i += 1) {
+      if (id.includes(i)) {
+        books.splice(i, 1);
+      }
+    }
+
+    storeForm(books);
+  });
+});
```

Comment:
- [ ] When click on the "Remove" button the book is not remove from the list of book nor from the storage unless we reload the page. And this is because the `deleteBtn` variable is not update when create a new book. You can fix it by reloading the page(`location.reload()`) whenever a book has been created.

Reply:
thank you